src/metabase/driver/mysql.clj

```diff
@@ -617,3 +620,65 @@
   [_driver]
   ;; https://dev.mysql.com/doc/refman/8.0/en/identifier-length.html
   64)
+
+(defn- sanitize-file-name
+  [name]
+  (str/replace name #"[^\w/\\.]" ""))
```

Comment:
This removes whitespace and hyphens that could be in the file-path. What is `sanitize-file-name` for anyway?